auth_helper.py

```diff
@@ -0,0 +1,68 @@
+import os
```

Comment:
This file should be in the `simple_ado` package. It should probably be something like `device_flow_helper` to be as explicit as possible.

Reply:
I'm fine with that change. Would we consider leaving it as auth_helper as it could potentially be the base for other authentication flows as well?

Reply:
I don't hugely mind the name, but I don't think it should be exposed to the users either way. I'd rather see two constructors for ADOClient, one which takes a PAT, one which takes a Tenant ID, App ID, etc. and calls into this class.

Reply:
Can you take the pull request as is and then I can look to make that change for the future?

Reply:
I don't think that's the right move. Temporary fixes like this shouldn't happen. If we merge and someone ends up depending on it, then fixing it would break things for them. We should do it right from the start.